src/styles/BLOCKS/card.scss

```
@@ -0,0 +1,95 @@
+.card {
+  box-sizing: border-box;
+  padding: 16px;
+  width: 200px;
+  margin-right: $space;
+
+  border: 1px solid #f3f3f3;
+
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+
+  &:nth-last-child(-n + 4) {
+    margin-top: 46px;
+  }
+
+  &__image {
+    margin: 16px 3px 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__product-image {
+    width: 160px;
+    height: 134px;
+  }
+
+  &__discription {
+    margin-bottom: 4px;
+    font-weight: 500;
+  }
+
+  &__code {
+    margin-bottom: 16px;
+
+    font-size: 10px;
+    line-height: 14px;
+    color: #616070;
+  }
+
+  &__feedback {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 24px;
+  }
+
+  &__responses {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    font-size: 10px;
+    line-height: 14px;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+  }
+
+  &__price-title {
+    margin-right: 55px;
+    color: #616070;
+  }
+
+  &__price-amount {
+    font-weight: 700;
+    font-size: 16px;
+  }
+
+  &__boughtLink {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 164px;
+    height: 38px;
+
+    text-transform: uppercase;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+    color: #fff;
+    text-decoration: none;
+
+    border: 1px solid #00acdc;
```

В тебе є цей колір у змінній
```suggestion
    border: 1px solid $blueColor
```